packages/rx_bloc_list/lib/src/extensions/identifiable_extensions.dart

```diff
@@ -109,27 +108,29 @@ extension ModelManageEvents<E extends Identifiable> on Stream<E> {
       _withLatestFromList(list).flatMap((tuple) async* {
         final identifiableInList = tuple.list
             .firstWhereOrNull((element) => element.id == tuple.item.id);
-
-        switch (await operationCallback(tuple.item, identifiableInList)) {
+        switch (await operationCallback(tuple.item)) {
           case ManageOperation.merge:
             yield ManagedList(
-              tuple.list._mergeWithList([tuple.item]),
```

Comment:
What is the reason of making those util methods public ?

Reply:
Dona